lib/util.js

```diff
@@ -159,7 +159,7 @@ function stylizeNoColor(str, styleType) {
 
 
 function arrayToHash(array) {
-  var hash = {};
+  var hash = Object.create(null);
 
   array.forEach(function(val, idx) {
     hash[val] = true;
```

Comment:
You'd check if `val` was `__proto__` here and discard it if it was, I think. cc @bnoordhuis